src/developer/web-api/tracker.md

```diff
@@ -2139,6 +2139,8 @@ This endpoint retrieves change logs for the attributes of a specific tracked ent
 |---|---|---|
 |path `/{uid}`|`String`|Tracked entity `UID`.|
 |`program`|`String`|Program `UID` (optional).|
+|`order`|`String`|Name and sort direction pair in the format `propName:sortDirection`.<br><br>Example: `createdAt:desc`<br><br>`propName` is case-sensitive. Valid sortDirection values are `asc` and `desc`. `sortDirection` is case-insensitive and defaults to `desc`. Supported properties are `attribute`, `createdAt`, and `username`, only one at a time.|
+|`filter`|`String`|Colon-separated field name with operator and value in the format `fieldName:operator:value`.<br><br>Example: `attribute:eq:w75KJ2mc4zz`<br><br>Filtering is supported for `attribute` and `username` fields, one at a time. Only the `eq` (equals) operator is supported.|
```

Comment:
I think we should settle on one of property or field. Maybe we should go for `field` as `field` is already part of our APIs via the parameter `field`.